deprecation.rst

```diff
@@ -0,0 +1,19 @@
+Deprecations
+============
+
+
+.. index:: EAPI; deprecated
+
+Deprecated EAPIs
+----------------
+:PG: 1001
+:Source: Council
```

Comment:
as far as I remember (from 2014 or so) the source of an EAPI deprecation was always first QA ond only then council :)

Reply:
Does it really matter though? The point is, people can't change it without Council decision, can they? All deprecations in layout.conf are attributed to Council.